pandas/tests/indexes/test_base.py

```diff
@@ -487,6 +491,7 @@ def test_constructor_cast(self):
         with pytest.raises(ValueError, match=msg):
             Index(["a", "b", "c"], dtype=float)
 
+    @pytest.mark.skipif(PY2, reason="pytest.raises match regex fails")
```

Comment:
Not a huge deal but we won't maintain PY2 support for the 0.25 release, so this will most likely subsequently be removed anyway

Reply:
that's right @WillAyd , we are just skipping for now and will remove this for 0.25, see https://github.com/pandas-dev/pandas/issues/23922#issuecomment-458897443